lib/generators/cdmdexer/settings_generator.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+require 'rails/generators'
+
+module Cdmdexer
+  class SettingsGenerator < Rails::Generators::Base
+    source_root ::File.expand_path('../templates', __FILE__)
+
+    desc <<-EOF
+      This generator makes the following changes to your application:
+       1. Installs settings.yml into your application's config directory
+    EOF
+
+    def install_settings
+      copy_file 'settings.yml', 'config/settings.yml'
```

Comment:
Can this be named `config/cdmdexer.yml`? settings.yml feels like it's headed for a name collision.

Reply:
Resolved in [9686827](https://github.com/UMNLibraries/cdmdexer/pull/8/commits/9686827a2b567951a25672f3d6088ed98f154684)